tinker/templates/faculty-bios/home.html

```diff
@@ -232,6 +258,64 @@ <h4 class="modal-title" id="myModalLabel">Are you sure you want to delete?</h4>
                 }
             });
         });
+
+
+        $(".courseleaf-user-press").click(function () {
```

Comment:
maybe change "press" to "click"? Just to keep in standard?

Reply:
changed press to click